registrations/templates/registrations/confirm_clinic.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
+
+{% block content %}
+<p> Is {{ clinic_name }} your facility? </p>
+<form method="post">
+    {% csrf_token %}
```

Comment:
what does this line do?

Reply:
This adds the CSRF (Cross-site request forgery) field to the form. You can read more about it here: https://docs.djangoproject.com/en/2.1/ref/csrf/

Basically, every time there's a GET request, it attaches a generated token to the form. That token has to be there, and be valid, on the POST request for us to accept it.

This is prevention against if a user has another site open in another tab in their browser, and that site makes a POST request to us, it will look like a valid POST request since it has the user's cookie, but it's not the user sending it. This prevents that, since the site won't know what the current CSRF token is.

Not sure if I explained it well, the docs do a better job.

Usually when you just do a `{{ form }}` in the template, it adds the CSRF token for you. Since we're constructing this manually here, we need to add the token ourselves.

By default Django will fail any POST request that doesn't have a valid CSRF token, unless you specifically tell Django that this view doesn't require a CSRF token